### PR TITLE
Generate the layer config of MLP to file

### PR DESCRIPTION
**Pull request by duchoang**
- [x] Generate layer config of MLP to text file
- [x] Ready for review

### CODE REVIEW

File: mlp/start_analysis.py [hunk 1]
```diff
@@ -5,16 +5,25 @@
 from training.acceleration_dataset import AccelerationDataset
 from training.mlp_model import MLPMeasurementModel
 
+def writeModel2File(layers, output_filename):
```
Comment (tmbo) [inline mlp/start_analysis.py]: python style is lower case with underscores for function names and CamelCase for class names

**Comment by tmbo** [comment]
I think the code should be moved out of `start_analysis` into `neon2iosmlp` since this is part of the conversion. This will also make clear that this only works for MLPs

**Comment by tmbo** [comment]
:+1:

**Comment by janm399** [comment]
Same naming comments as #12. Once resolved, OK to merge.

**Comment by duchoang** [comment]
This PR https://github.com/muvr/muvr-analytics/pull/12 will resolve the naming convention.

**Pull request merged**